.changeset/lazy-seahorses-share.md

```diff
@@ -0,0 +1,13 @@
+---
+"@itwin/itwinui-react": major
```

Comment:
This changeset should be on the css package, not react.

Reply:
```suggestion
"@itwin/itwinui-css": major
```